pkg/cloud/services/compute/instance_test.go

```diff
@@ -48,6 +49,11 @@ func Test_getPortName(t *testing.T) {
 			args: args{"test-1-instance", &infrav1.PortOpts{}, 4},
 			want: "test-1-instance-4",
 		},
+		{
+			name: "with PortOpts name suffix",
+			args: args{"test-1-instance", &infrav1.PortOpts{NameSuffix: "foo2", NetworkID: "bar", DisablePortSecurity: &True}, 4},
```

Comment:
```suggestion
			args: args{"test-1-instance", &infrav1.PortOpts{NameSuffix: "foo2", NetworkID: "bar", DisablePortSecurity: &iTrue}, 4},
```

Reply:
`pull-cluster-api-provider-openstack-test` is failing due to that.

Reply:
Ack, yeah it doesn't like the style. I think doing ```var x = y``` is also bad style and should be ```x := y```